microproxy/config.py

```diff
@@ -160,4 +189,10 @@ def parse_config():
 
     ini_parser = ConfigParserBuilder.setup_ini_parser()
     ini_parser.read(config_file)
-    return Config(ini_parser, vars(cmd_config))
+
+    config = Config(ini_parser, vars(cmd_config))
+
+    if is_config_correct(config):
+        return config
+    else:
+        return None
```

Comment:
I think we should raise Error with the reason when config is incorrect.
Because if we return None here and try to run program and will still failed or exit without the reason why failed.